pkg/translator/faro/traces_to_faro_test.go

```diff
@@ -0,0 +1,201 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package faro // import "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/faro"
+
+import (
+	"context"
+	"encoding/hex"
+	"testing"
+
+	faroTypes "github.com/grafana/faro/pkg/go"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.21.0"
+)
+
+var emptyScopeSpansAndEmptyResourceAttributes = func() ptrace.Traces {
+	td := ptrace.NewTraces()
+	td.ResourceSpans().AppendEmpty()
+	return td
+}
+
+var twoResourceSpansWithDifferentServiceNameResourceAttribute = func(t *testing.T) ptrace.Traces {
```

Comment:
Expected values should be in golden files.

Reply:
fixed